bolt-gcc-stage2.bash

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+export TOPLEV=~/toolchain/gcc
+mkdir ${TOPLEV}
+cd ${TOPLEV}
+
+mkdir -p ${TOPLEV}/bolt-gcc/intrumentdata || (echo "Could not create stage3-bolt directory"; exit 1)
+cd ${TOPLEV}/bolt-gcc
+GCCPATH=/usr/lib/gcc/x86_64-pc-linux-gnu/12.1.1
```

Comment:
same as other script.
additionally, it could be extracted into a common file that is sourced.

Duplication is the devil